modules/supporting_encapsulation.adoc

```diff
@@ -0,0 +1,93 @@
+  | `-/sys/fs/fuse/connections                                                                                                                             fusectl                fusectl    rw,relatime
+  |-/dev                                                                                                                                                   devtmpfs               devtmpfs   rw,nosuid,seclabel,size=16317680k,nr_inodes=4079420,mode=755
+  | |-/dev/shm                                                                                                                                             tmpfs                  tmpfs      rw,nosuid,nodev,seclabel
+  | |-/dev/pts                                                                                                                                             devpts                 devpts     rw,nosuid,noexec,relatime,seclabel,gid=5,mode=620,ptmxmode=000
+  | |-/dev/mqueue                                                                                                                                          mqueue                 mqueue     rw,relatime,seclabel
+  | `-/dev/hugepages                                                                                                                                       hugetlbfs              hugetlbfs  rw,relatime,seclabel,pagesize=2M
+  |-/run                                                                                                                                                   tmpfs                  tmpfs      rw,nosuid,nodev,seclabel,mode=755
+  | |-/run/containers/storage/overlay-containers/35f65a7e081f135c3809350240e94cf6113b6e283f34191f8a5976041a52430e/userdata/shm                             shm                    tmpfs      rw,nosuid,nodev,noexec,relatime,context="system_u:object_r:container_file_t:s0:c284,c530",size=65536k
+  | |-/run/utsns/f6f4d4cd-a701-462b-87dd-ec645b49f6b0                                                                                                      nsfs[uts:[4026532370]] nsfs       rw,seclabel
+  | |-/run/ipcns/f6f4d4cd-a701-462b-87dd-ec645b49f6b0                                                                                                      nsfs[ipc:[4026532371]] nsfs       rw,seclabel
+  | |-/run/netns/f6f4d4cd-a701-462b-87dd-ec645b49f6b0                                                                                                      nsfs[net:[4026531992]] nsfs       rw,seclabel
+  | |-/run/containers/storage/overlay-containers/ec566c5b5e84eebe1f2e0f9259ac0096e87f6e49041f3f136379f7efbe4e74f2/userdata/shm                             shm                    tmpfs      rw,nosuid,nodev,noexec,relatime,context="system_u:object_r:container_file_t:s0:c294,c852",size=65536k
+  | |-/run/containers/storage/overlay-containers/ba6dc1dd63e5078fdb46b4a492b2172d63b10d26a4e8fb613ab301658ef3d1b2/userdata/shm                             shm                    tmpfs
```

Comment:
This is not the full output.  The full output on an OpenShift cluster is very very long.  Should we add an ellipsis to indicate there's more they would see in the real world?
```suggestion
  | |-/run/containers/storage/overlay-containers/ba6dc1dd63e5078fdb46b4a492b2172d63b10d26a4e8fb613ab301658ef3d1b2/userdata/shm                             shm                    tmpfs
...
```

Reply:
Yes, let's go with the ellipse.